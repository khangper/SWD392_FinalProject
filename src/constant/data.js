export const API_KEY = "API_KEY";

export const FETCH_HOME_LIVESTREAM_REQUEST = "FETCH_HOME_LIVESTREAM_REQUEST";
export const FETCH_HOME_LIVESTREAM_SUCCESS = "FETCH_HOME_LIVESTREAM_SUCCESS";
export const FETCH_HOME_LIVESTREAM_FAILURE = "FETCH_HOME_LIVESTREAM_FAILURE";

export const FETCH_HOME_FEATURED_COURSES_REQUEST =
  "FETCH_HOME_FEATURED_COURSES_REQUEST";
export const FETCH_HOME_FEATURED_COURSES_SUCCESS =
  "FETCH_HOME_FEATURED_COURSES_SUCCESS";
export const FETCH_HOME_FEATURED_COURSES_FAILURE =
  "FETCH_HOME_FEATURED_COURSES_FAILURE";
// All Live Stream
export const FETCH_STREAMS_REQUEST = "FETCH_STREAMS_REQUEST";
export const FETCH_STREAMS_SUCCESS = "FETCH_STREAMS_SUCCESS";
export const FETCH_STREAMS_FAILURE = "FETCH_STREAMS_FAILURE";

//All Instructor
export const FETCH_INSTRUCTORS_REQUEST = "FETCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTORS_SUCCESS = "FETCH_INSTRUCTORS_SUCCESS";
export const FETCH_INSTRUCTORS_FAILURE = "FETCH_INSTRUCTORS_FAILURE";
export const SEARCH_INSTRUCTORS_REQUEST = "SEARCH_INSTRUCTORS_REQUEST";
export const FETCH_INSTRUCTOR_BY_ID_REQUEST = "FETCH_INSTRUCTOR_BY_ID_REQUEST";

// SaveCourse.js
export const FETCH_SAVECOURSES_REQUEST = "FETCH_SAVECOURSES_REQUEST";
export const FETCH_SAVECOURSES_SUCCESS = "FETCH_SAVECOURSES_SUCCESS";
export const FETCH_SAVECOURSES_FAILURE = "FETCH_SAVECOURSES_FAILURE";
export const DELETE_SAVECOURSE_REQUEST = "DELETE_SAVECOURSE_REQUEST";
export const DELETE_SAVECOURSE_SUCCESS = "DELETE_SAVECOURSE_SUCCESS";
export const DELETE_SAVECOURSE_FAILURE = "DELETE_SAVECOURSE_FAILURE";

// Shoppingcart.js
export const ShoppingCart_API_URL =
  "https://66908916c0a7969efd9c67ed.mockapi.io/ojt-repo/ShoppingCart";
export const ADD_TO_CART = "ADD_TO_CART";
export const REMOVE_FROM_CART = "REMOVE_FROM_CART";
export const APPLY_DISCOUNT = "APPLY_DISCOUNT";
export const ADD_TO_CART_FAILURE = " ADD_TO_CART_FAILURE";
export const ADD_TO_CART_SUCCESS = "APPLY_DISCOUNT";

//MyCourse_Course
export const FETCH_COURSES_REQUEST = "FETCH_COURSES_REQUEST";
export const FETCH_COURSES_SUCCESS = "FETCH_COURSES_SUCCESS";
export const FETCH_COURSES_FAILURE = "FETCH_COURSES_FAILURE";

export const EDIT_COURSE_REQUEST = "EDIT_COURSE_REQUEST";
export const EDIT_COURSE_SUCCESS = "EDIT_COURSE_SUCCESS";
export const EDIT_COURSE_FAILURE = "EDIT_COURSE_FAILURE";

export const DELETE_COURSE_REQUEST = "DELETE_COURSE_REQUEST";
export const DELETE_COURSE_SUCCESS = "DELETE_COURSE_SUCCESS";
export const DELETE_COURSE_FAILURE = "DELETE_COURSE_FAILURE";

//UpcommingCourse_Course
export const FETCH_UPCOMMINGCOURSES_REQUEST = "FETCH_UPCOMMINGCOURSES_REQUEST";
export const FETCH_UPCOMMINGCOURSES_SUCCESS = "FETCH_UPCOMMINGCOURSES_SUCCESS";
export const FETCH_UPCOMMINGCOURSES_FAILURE = "FETCH_UPCOMMINGCOURSES_FAILURE";

export const EDIT_UPCOMMINGCOURSE_REQUEST = "EDIT_UPCOMMINGCOURSE_REQUEST";
export const EDIT_UPCOMMINGCOURSE_SUCCESS = "EDIT_UPCOMMINGCOURSE_SUCCESS";
export const EDIT_UPCOMMINGCOURSE_FAILURE = "EDIT_UPCOMMINGCOURSE_FAILURE";

export const DELETE_UPCOMMINGCOURSE_REQUEST = "DELETE_UPCOMMINGCOURSE_REQUEST";
export const DELETE_UPCOMMINGCOURSE_SUCCESS = "DELETE_UPCOMMINGCOURSE_SUCCESS";
export const DELETE_UPCOMMINGCOURSE_FAILURE = "DELETE_UPCOMMINGCOURSE_FAILURE";
