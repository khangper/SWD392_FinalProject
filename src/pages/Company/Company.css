/*Expanding learning opportunities*/

.company-about,
.company-blog,
.company-careers,
.company-company,
.company-press {
  position: relative;
  line-height: 20px;
  font-weight: 550;
  color: #424242;
  display: inline-block;
}

.company-about {
  min-width: 38px;
}
.company-blog {
  min-width: 29px;
}
.company-company {
  min-width: 60px;
}
.company-careers {
  min-width: 49px;
}
.company-press {
  min-width: 36px;
}

.company-navigation-links-1,
.company-navigation-links-2,
.company-navigation-links-3,
.company-navigation-links-4 {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: flex-start;
}

.company-navigation-links {
  padding: 6px 0 0;
  margin-right: 18px;
}

.company-navigation-links-4 {
  padding: 6px 0 0;
}

.company-navigation-links-1,
.company-navigation-links-3 {
  padding: 6px 20px 0 0;
}

.company-navigation-links-2 {
  border-bottom: 2px solid #ed2a26;
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-start;
  margin-left: -20px;
  padding: 6px 20px 7px 21px;
}

.company-navigation-links-parent {
  display: flex;
  align-items: flex-start;
  justify-content: flex-start;
  flex-direction: row;
  gap: 14px;
  max-width: 100%;
  font-size: 14px;
}

.company-heading-2 {
  margin: 0;
  position: relative;
  font-size: 24px;
  line-height: 30px;
  font-weight: 600;
  font-family: inherit;
  color: #333;
  flex-shrink: 0;
}

.company-learning-heading {
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-end;
  padding: 0 44px 0 47px;
  box-sizing: border-box;
  max-width: 100%;
  text-align: center;
  font-size: 24px;
}

.company-background {
  align-self: stretch;
  background-image: url("../../assets/background_about.jpg");
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
  padding: 30px 20px 55px;
  box-sizing: border-box;
  position: relative;
  gap: 65px;
  min-height: 216px;
  max-width: 100%;
  text-align: left;
  font-size: 14px;
  color: #333;
}

/*Our Story*/
.company-heading-23 {
  margin: 0;
  position: relative;
  font-size: 22px;
  line-height: 26px;
  font-weight: 600;
  font-family: inherit;
  display: inline-block;
  min-width: 102.2px;
}

.company-line-story {
  border-bottom: 2px solid #ed2a26;
  width: 50px;
  align-self: stretch;
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: center;
}

.company-heading-2-our-story {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: flex-start;
  gap: 20px;
}

.company-our-story-content {
  margin: 0;
}

.company-our-story-container {
  position: relative;
  font-size: 14px;
  line-height: 26px;
  color: #686f7a;
}

.company-story-content,
.company-story-wrapper {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: flex-start;
}

.company-story-wrapper {
  gap: 33px;
}

.company-story-content {
  padding: 0 0 46px;
  box-sizing: border-box;
  min-width: 573.9px;
  max-width: 100%;
  width: 600px;
}

.company-story-img {
  width: 435px;
  position: relative;
  max-height: 100%;
  overflow: hidden;
  flex-shrink: 0;
  object-fit: cover;
  max-width: 100%;
}

.company-our-story {
  align-self: stretch;
  background-color: #fff;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: center;
  padding-top: 62px;
  padding-bottom: 62px;
  box-sizing: border-box;
  gap: 70px;
  max-width: 100%;
  flex-shrink: 0;
  text-align: left;
  font-size: 24px;
  color: #333;
}

/*Office Location*/
.company-line-wrapper {
  border-bottom: 2px solid #ed2a26;
  width: 50px;
  padding-top: 20px;
}

.company-heading-3 {
  margin: 0;
  font-size: 22px;
  position: relative;
  line-height: 26px;
  font-weight: 600;
  font-family: inherit;
  display: inline-block;
  min-width: 122px;
  color: #333;
}

.company-location-details-one {
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-start;
}

.company-cursus-branches-around {
  position: relative;
  line-height: 26px;
}

.company-location-elements-one,
.company-office-one-description {
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: flex-start;
}

.company-office-one-description {
  align-self: stretch;
  font-style: 16px;
  color: #686f7a;
}

.company-location-elements-one {
  gap: 17px;
}

.company-office-location-one {
  align-self: stretch;
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: center;
  padding: 0 20px 0 21px;
}

.company-image-company {
  height: 200px;
  width: 340px;
  position: absolute;
  margin: 0 !important;
  top: 0;
  right: 0;
  left: 0;
  bottom: 0;
  border-radius: 3px;
  max-width: 100%;
  max-height: 100%;
  object-fit: cover;
}

.company-gradient,
.company-link-2 {
  align-self: stretch;
  position: relative;
  max-width: 100%;
}

.company-gradient {
  /* background: linear-gradient(180deg, transparent, rgba(51, 51, 51, 0.3) 90%); */
  z-index: 1;
  width: 340px;
  height: 200px;
}

.company-link-2 {
  display: flex;
  flex-direction: row;
  align-items: flex-start;
  justify-content: flex-start;
}

.company-country {
  position: relative;
  font-size: 16px;
  line-height: 26px;
  font-weight: 550;
  color: #333;
  display: inline-block;
  min-width: 94px;
}

.company-address {
  margin: 0;
}
.company-address-container {
  position: relative;
  line-height: 26px;
}

.company-icon-phone {
  width: 14px;
  height: 14px;
  position: relative;
  flex-shrink: 0;
  color: #686f7a;
}

.company-icon-phone:hover{
    color: #333;
}

.company-office-two-title {
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: flex-end;
  padding: 0 0 5px;
}

.company-office-two-address {
  position: relative;
  line-height: 26px;
  display: inline-block;
  min-width: 96.4px;
  flex-shrink: 0;
}

.company-location-details-two {
  display: flex;
  flex-direction: row;
  align-items: flex-end;
  justify-content: flex-start;
  gap: 10px;
}

.company-location-details-two:hover{
    color: #333;
    cursor: pointer;
}

.company-location-elements-two {
  flex-direction: row;
  padding: 0 2px;
  font-family: inherit;
}

.company-background-border,
.company-location-elements-two,
.company-office-city-one,
.company-office-full-address-one {
  display: flex;
  align-items: flex-start;
  justify-content: flex-start;
}

.company-office-full-address-one {
  flex-direction: column;
  gap: 15px;
  padding-top: 20px;
  width: 100%;
}

.company-background-border,
.company-office-city-one {
  box-sizing: border-box;
}

.company-office-city-one {
  flex-direction: row;
  max-width: 100%;
}

.company-background-border {
  height: 386.8px;
  flex: 1;
  border-radius: 3px;
  background-color: #fff;
  border: 1px solid #efefef;
  flex-direction: column;
  padding: 11px 10px 21.5px 14px;
  max-width: calc(100% - 800px);
  text-align: left;
  font-size: 14px;
  color: #686f7a;
  font-family: inherit;
}

.company-office-link-one,
.company-office-location-one-parent{
    display: flex;
    
    max-width: 100%;
}

.company-office-link-one{
    align-self: stretch;
    flex-direction: row;
    align-items: flex-start;
    padding: 0 340px;
    box-sizing: border-box;
    gap: 21px;
    text-align: left;
    font-size: 14px;
    color: #686f7a;
    justify-content: space-between;
}

.company-office-location-one-parent{
    align-self: stretch;
    flex-direction: column;
    align-items: flex-start;
    padding: 50px 20px;
    box-sizing: border-box;
    text-align: center;
    color: #333;
    font-family: inherit;
    height: 650px;
    background-color: #f7f7f7;
    justify-content: flex-start;
    gap: 41px
}

@media screen and (max-width: 1630px) {

  .company-office-link-one{
    padding-left: 200px;
    padding-right: 200px ;
  }

  .company-our-story{
    flex-wrap: wrap;
  }

}

@media screen and (max-width: 1350px){
    .company-background-border{
        max-width: 100%;
        display: none;
    }

    .company-story-content{
        padding: 0;
    }

    .company-our-story{
        gap: 42px;
        padding: 0 185px 40px;
        box-sizing: border-box;
    }

}

@media screen and (max-width: 900px){
    .company-office-location-one-parent{
        gap: 20px;
    }

    .company-story-content{
        gap: 16px;
    }

    .company-our-story{
        gap: 21px;
        padding: 0 92px;
        box-sizing: border-box;
    }
}

@media screen and (max-width: 450px){
    .company-heading-3{
        font-size: 19px;
        line-height: 21px;
    }

    .company-heading-23{
        font-size: 19px;
        line-height: 21px;
    }

    .company-our-story{
        padding: 40px 20px;
        box-sizing: border-box;
    }
}