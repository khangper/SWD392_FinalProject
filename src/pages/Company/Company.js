import React from "react";
import "./Company.css";
import Header from "../../components/Header/Header";
import India from "../../assets/India.png";
import Cali from "../../assets/Cali.png";
import Brazil from "../../assets/Brazil.png";
import { FiPhone } from "react-icons/fi";
import Footer from "../../components/Footer/Footer";

const Company = () => {
  return (
    <div className="company">
      <Header />
      <main>
        <div className="company-parent">
          <div className="company-background">
            <div className="company-navigation-links-parent">
              <div className="company-navigation-links">
                <a href="/about-us" className="company-about">
                  About
                </a>
              </div>
              <div className="company-navigation-links-1">
                <a href="/our-blog" className="company-blog">
                  Blog
                </a>
              </div>
              <div className="company-navigation-links-2">
                <a href="/company-details" className="company-company">
                  Company
                </a>
              </div>
              <div className="company-navigation-links-3">
                <a href="/career" className="company-careers">
                  Careers
                </a>
              </div>
              <div className="company-navigation-links-4">
                <a href="/press" className="company-press">
                  Press
                </a>
              </div>
            </div>
            <div className="company-learning-heading">
              <h2 className="company-heading-2">
                Expanding learning opportunities
              </h2>
            </div>
          </div>

          <div className="company-our-story">
            <div className="company-story-content">
              <div className="company-story-wrapper">
                <div className="company-heading-2-our-story">
                  <h2 className="company-heading-23">Our Story</h2>
                  <div className="company-line-story"></div>
                </div>
                <div className="company-our-story-container">
                  <p className="company-our-story-content">
                    Cursus was founded in 2020 by computer science instructor
                    with a vision to provide anyone, anywhere with access to the
                    worldâ€™s best education. Now many instructors put their
                    courses online for anyone to take and taught more learners
                    in a few months than they could over an entire lifetime in
                    the classroom. Today, Cursus has expanded to reach more than
                    40 million people and 2,300 businesses around the world. On
                    Cursus you can find online courses, instructors, and
                    certificates from Cursus.
                  </p>
                </div>
              </div>
            </div>
            <img
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/about/company.jpg"
              alt=""
              className="company-story-img"
            />
          </div>

          <div className="company-office-locations">
            <div className="company-office-location-one-parent">
              <div className="company-office-location-one">
                <div className="company-location-elements-one">
                  <div className="company-location-details-one">
                    <h3 className="company-heading-3">Our Offices</h3>
                  </div>
                  <div className="company-office-one-description">
                    <div className="company-cursus-branches-around">
                      Cursus branches around the world
                    </div>
                    <div className="company-line-wrapper"></div>
                  </div>
                </div>
              </div>
              <div className="company-office-link-one">
                <div className="company-background-border">
                  <div className="company-link-2">
                    <img src={India} alt="" className="company-image-company" />
                    <div className="company-gradient" />
                  </div>
                  <div className="company-office-city-one">
                    <div className="company-office-full-address-one">
                      <div className="company-country">Punjab, India</div>
                      <div className="company-address-container">
                        <p className="company-address">
                          #1235 Sks Nagar St No. 8 Phase 3, Pakhowal Road,
                          141001, LDH, Punjab, India
                        </p>
                      </div>
                      <div className="company-location-elements-two">
                        <div className="company-location-details-two">
                          <div className="company-office-two-title">
                            <FiPhone className="company-icon-phone" />
                          </div>
                          <div className="company-office-two-address">
                            0161-1234567
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>

                <div className="company-background-border">
                  <div className="company-link-2">
                    <img src={India} alt="" className="company-image-company" />
                    <div className="company-gradient" />
                  </div>
                  <div className="company-office-city-one">
                    <div className="company-office-full-address-one">
                      <div className="company-country">Punjab, India</div>
                      <div className="company-address-container">
                        <p className="company-address">
                          #1235 Sks Nagar St No. 8 Phase 3, Pakhowal Road,
                          141001, LDH, Punjab, India
                        </p>
                      </div>
                      <div className="company-location-elements-two">
                        <div className="company-location-details-two">
                          <div className="company-office-two-title">
                            <FiPhone className="company-icon-phone" />
                          </div>
                          <div className="company-office-two-address">
                            0161-1234567
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>

                <div className="company-background-border">
                  <div className="company-link-2">
                    <img src={India} alt="" className="company-image-company" />
                    <div className="company-gradient" />
                  </div>
                  <div className="company-office-city-one">
                    <div className="company-office-full-address-one">
                      <div className="company-country">Punjab, India</div>
                      <div className="company-address-container">
                        <p className="company-address">
                          #1235 Sks Nagar St No. 8 Phase 3, Pakhowal Road,
                          141001, LDH, Punjab, India
                        </p>
                      </div>
                      <div className="company-location-elements-two">
                        <div className="company-location-details-two">
                          <div className="company-office-two-title">
                            <FiPhone className="company-icon-phone" />
                          </div>
                          <div className="company-office-two-address">
                            0161-1234567
                          </div>
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </main>

      <Footer />
    </div>
  );
};

export default Company;
