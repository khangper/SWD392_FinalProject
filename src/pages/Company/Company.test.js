import React from "react";
import { render, screen } from "@testing-library/react";
import Company from "../Company/Company";
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("Company", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the Company component", async () => {
    const { container } = renderWithProviders(<Company />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});
