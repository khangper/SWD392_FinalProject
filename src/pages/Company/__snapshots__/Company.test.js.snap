// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Company renders the Company component 1`] = `
<div>
  <div
    class="company"
  >
    <header
      class="header"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <div
        class="profile"
      >
        <img
          alt=""
          class="header-avatar"
          src="profile-img.jpg"
        />
      </div>
    </header>
    <main>
      <div
        class="company-parent"
      >
        <div
          class="company-background"
        >
          <div
            class="company-navigation-links-parent"
          >
            <div
              class="company-navigation-links"
            >
              <a
                class="company-about"
                href="/about-us"
              >
                About
              </a>
            </div>
            <div
              class="company-navigation-links-1"
            >
              <a
                class="company-blog"
                href="/our-blog"
              >
                Blog
              </a>
            </div>
            <div
              class="company-navigation-links-2"
            >
              <a
                class="company-company"
                href="/company-details"
              >
                Company
              </a>
            </div>
            <div
              class="company-navigation-links-3"
            >
              <a
                class="company-careers"
                href="/career"
              >
                Careers
              </a>
            </div>
            <div
              class="company-navigation-links-4"
            >
              <a
                class="company-press"
                href="/press"
              >
                Press
              </a>
            </div>
          </div>
          <div
            class="company-learning-heading"
          >
            <h2
              class="company-heading-2"
            >
              Expanding learning opportunities
            </h2>
          </div>
        </div>
        <div
          class="company-our-story"
        >
          <div
            class="company-story-content"
          >
            <div
              class="company-story-wrapper"
            >
              <div
                class="company-heading-2-our-story"
              >
                <h2
                  class="company-heading-23"
                >
                  Our Origins
                </h2>
                <div
                  class="company-line-story"
                />
              </div>
              <div
                class="company-our-story-container"
              >
                <p
                  class="company-our-story-content"
                >
                  Cursus was founded in 2020 by computer science instructor with a vision to provide anyone, anywhere with access to the worldâ€™s best education. Now many instructors put their courses online for anyone to take and taught more learners in a few months than they could over an entire lifetime in the classroom. Today, Cursus has expanded to reach more than 40 million people and 2,300 businesses around the world. On Cursus you can find online courses, instructors, and certificates from Cursus.
                </p>
              </div>
            </div>
          </div>
          <img
            alt=""
            class="company-story-img"
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/about/company.jpg"
          />
        </div>
        <div
          class="company-office-locations"
        >
          <div
            class="company-office-location-one-parent"
          >
            <div
              class="company-office-location-one"
            >
              <div
                class="company-location-elements-one"
              >
                <div
                  class="company-location-details-one"
                >
                  <h3
                    class="company-heading-3"
                  >
                    Our Offices
                  </h3>
                </div>
                <div
                  class="company-office-one-description"
                >
                  <div
                    class="company-cursus-branches-around"
                  >
                    Cursus branches around the world
                  </div>
                  <div
                    class="company-line-wrapper"
                  />
                </div>
              </div>
            </div>
            <div
              class="company-office-link-one"
            >
              <div
                class="company-background-border"
              >
                <div
                  class="company-link-2"
                >
                  <img
                    alt=""
                    class="company-image-company"
                    src="India.png"
                  />
                  <div
                    class="company-gradient"
                  />
                </div>
                <div
                  class="company-office-city-one"
                >
                  <div
                    class="company-office-full-address-one"
                  >
                    <div
                      class="company-country"
                    >
                      Punjab, India
                    </div>
                    <div
                      class="company-address-container"
                    >
                      <p
                        class="company-address"
                      >
                        #1235 Sks Nagar St No. 8 Phase 3, Pakhowal Road, 141001, LDH, Punjab, India
                      </p>
                    </div>
                    <div
                      class="company-location-elements-two"
                    >
                      <div
                        class="company-location-details-two"
                      >
                        <div
                          class="company-office-two-title"
                        >
                          <svg
                            class="company-icon-phone"
                            fill="none"
                            height="1em"
                            stroke="currentColor"
                            stroke-linecap="round"
                            stroke-linejoin="round"
                            stroke-width="2"
                            viewBox="0 0 24 24"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                            />
                          </svg>
                        </div>
                        <div
                          class="company-office-two-address"
                        >
                          0161-1234567
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="company-background-border"
              >
                <div
                  class="company-link-2"
                >
                  <img
                    alt=""
                    class="company-image-company"
                    src="Cali.png"
                  />
                  <div
                    class="company-gradient"
                  />
                </div>
                <div
                  class="company-office-city-one"
                >
                  <div
                    class="company-office-full-address-one"
                  >
                    <div
                      class="company-country"
                    >
                      San Francisco, CA
                    </div>
                    <div
                      class="company-address-container"
                    >
                      <p
                        class="company-address"
                      >
                        586 Lorem st. 5 floor, San Francisco San Francisco, CA 94107
                      </p>
                    </div>
                    <div
                      class="company-location-elements-two"
                    >
                      <div
                        class="company-location-details-two"
                      >
                        <div
                          class="company-office-two-title"
                        >
                          <svg
                            class="company-icon-phone"
                            fill="none"
                            height="1em"
                            stroke="currentColor"
                            stroke-linecap="round"
                            stroke-linejoin="round"
                            stroke-width="2"
                            viewBox="0 0 24 24"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                            />
                          </svg>
                        </div>
                        <div
                          class="company-office-two-address"
                        >
                          +1 415-1234567
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="company-background-border"
              >
                <div
                  class="company-link-2"
                >
                  <img
                    alt=""
                    class="company-image-company"
                    src="Brazil.png"
                  />
                  <div
                    class="company-gradient"
                  />
                </div>
                <div
                  class="company-office-city-one"
                >
                  <div
                    class="company-office-full-address-one"
                  >
                    <div
                      class="company-country"
                    >
                      SÃ£o Paulo, Brazil
                    </div>
                    <div
                      class="company-address-container"
                    >
                      <p
                        class="company-address"
                      >
                        Lorem ipsum 589, Vila Madalena, SÃ£o Paulo - SP 01443-010
                      </p>
                    </div>
                    <div
                      class="company-location-elements-two"
                    >
                      <div
                        class="company-location-details-two"
                      >
                        <div
                          class="company-office-two-title"
                        >
                          <svg
                            class="company-icon-phone"
                            fill="none"
                            height="1em"
                            stroke="currentColor"
                            stroke-linecap="round"
                            stroke-linejoin="round"
                            stroke-width="2"
                            viewBox="0 0 24 24"
                            width="1em"
                            xmlns="http://www.w3.org/2000/svg"
                          >
                            <path
                              d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                            />
                          </svg>
                        </div>
                        <div
                          class="company-office-two-address"
                        >
                          +55-11-1234567
                        </div>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                â–¼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          Â© 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
